### ISSUE TITLE
Ladda's CSS must be made part of react-ladda package

### ISSUE THREAD

**Issue by InamTaj** [jsdir/react-ladda#51]
As this package is intended for react-apps, the [css file](https://cdnjs.cloudflare.com/ajax/libs/Ladda/1.0.0/ladda.min.css) for ladda buttons must be included in the **react-ladda** package so that one can perform clean imports / builds in projects that use webpack + [css-modules](https://github.com/css-modules/css-modules) for managing CSS.

Example Usage from my app's `App.js`:
```
...
// imports of CSS styles of 3rd-party packages
import 'font-awesome/css/font-awesome.min.css';
import 'rc-slider/assets/index.css';
import 'react-datepicker/dist/react-datepicker.css';
import 'react-select/dist/react-select.css';
```

**Comment by austinbutler** [comment]
I think you can still import the CSS from Ladda itself from its `dist` folder, since it gets installed with `react-ladda`.

**Comment by InamTaj** [comment]
@austinbutler I've already checked and there is no style available in the `dist` folder.

**Comment by austinbutler** [comment]
I am referring to the `ladda` module itself, not `react-ladda`.

![image](https://user-images.githubusercontent.com/354741/43800672-e05f1234-9a45-11e8-8cc3-388af2e53b5e.png)

**Comment by InamTaj** [comment]
@austinbutler Yes this is what I did eventually. But the issue I opened is that the style should be included in the `react-ladda` so that it's easier to use.
